keepassxc-browser/_locales/en/messages.json

```diff
@@ -683,6 +687,10 @@
         "message": "Show a dropdown list containing available credentials for all username fields on a page.",
         "description": "Autocomplete Usernames option help text."
     },
+    "optionsShowGroupNameInAutocompleteHelpText": {
+        "message": "This only happens if there are matching credentials from different groups for a page.",
```

Comment:
Recommend: "When credentials are returned from different groups, the group name will be shown."